### DISCUSSION TITLE
Incognito [Legacy] Doesn't work on v122-

### DISCUSSION THREAD

**Discussion by ceemeck** [3kh0/ext-remover#1304] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
The exploit needs the flag called chrome://flags/#captive-portal-popup-window, this is not in v122 or older, so it isn't possible. Basically a minor bug, not sure where to report bugs anyways but I decided it was worth saying something since it's a bit misleading

**Comment by S-PScripts** [comment]
ok
